cmd/collectors/simple/simple.go

```diff
@@ -104,6 +106,7 @@ func (n *NodeMon) PollInstance() (*matrix.Matrix, error) {
 		}
 		instance.SetLabel("poller", name)
 		instance.SetLabel("version", version.VERSION)
+		instance.SetLabel("pid", strconv.Itoa(os.Getpid()))
```

Comment:
does it work when started as root user (bin/harvest start) in rpms?

Reply:
it should since it's Go's code, not the ps code